lib/functions.php

```diff
@@ -215,6 +218,16 @@ function ($matches) {
         }
     }
 
+    /*
+     * phpstan is not able to process all the things that happen while this function
+     * constructs the result array. It only understands the $result is
+     * non-empty-array<string, mixed>
+     *
+     * But the detail of the returned array is correctly specified in the PHPdoc
+     * above the function call.
+     *
+     * @phpstan-ignore-next-line
+     */
     return
          $result + [
```

Comment:
I guess my recent fix regarding `+` will resolve this phpstan error on the next release

https://github.com/phpstan/phpstan-src/pull/1579